src/invisible/how-tos/advanced-analysis/intrinsics/README.ipynb

```diff
@@ -133,11 +133,13 @@
     "    (200., 500.),\n",
     "    (900., 900.),\n",
     "])\n",
+    "# Define a different color for each point\n",
```

Comment:
For this cell the objective is to use as few lines as possible to get the points across. IMO the additional complexity for varying colors does not help this objective.